applications/decentralized-pallet-staking-audit-tool.md

```diff
@@ -0,0 +1,374 @@
+| **0a.** | License                     | Dual Apache 2.0, MIT |
+| **0b.** | Documentation               | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide   | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker                      | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1.a     | Finalized user guide: Dashboard  | Dashboard descriptions. English language, markdown format (Overleaf). |
+| 1.b     | Finalized user guide: Data ETL   | Data script descriptions. English language, markdown format (Overleaf). |
+| 1.c     | Finalized user guide: Staking    | Staking descriptions. English language, markdown format (Overleaf). |
+| 1.d     | Finalized user guide: Dilution   | Dilution descriptions. English language, markdown format (Overleaf). |
+| 1.e     | Finalized user guide: Testing    | Test script descriptions. English language, markdown format (Overleaf). |
+| 1.f     | Finalized workbook: Dashboard    | Workbook dashboard sheets structure. English language, OpenOffice/LibreOffice format. |
+| 1.g     | Finalized workbook: Data         | Workbook data sheets structure. English language, OpenOffice/LibreOffice format. |
+| 1.h     | Finalized workbook: Staking      | Workbook staking logic. English language, OpenOffice/LibreOffice format. |
+| 1.i     | Finalized workbook: Dilution     | Workbook dilution logic. English language, OpenOffice/LibreOffice format. |
+| 1.j     | Finalized workbook: Charts       | Workbook charts. English language, OpenOffice/LibreOffice format. |
+| 1.k     | Documentation issues & PRs  | Open and engage in documentation related issues and PRs. The scope is to align any documentation with the code. |
+| 1.l     | Implementation issues & PRs | Open and engage in implementation related issues and PRs. These will be limited to test cases and code comments.|
+| 1.m     | Staking Pallet PR           | Open and engage in PR to add this tool to the Staking Pallet. |
+
+### Milestone 4 — Final milestone review, scope creep
+
+- **Estimated duration:** 1 month
+- **FTE:**  0.5 FTE
+- **Costs:** 10,000 USD
+
+| Number  | Deliverable                      | Specification |
+| -----:  | -----------                      | ------------- |
+| **0a.** | License                          | Dual Apache 2.0, MIT |
+| **0b.** | Documentation                    | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
```

Comment:
Why do you need to send transactions or run a Substrate node for this? If you are relaying on Subscan (which it looks like you are elsewhere), you are already dependent on their nodes.